frontend/src/metabase-types/store/mocks/embed.ts

```diff
@@ -1,12 +1,13 @@
 import { EmbedOptions, EmbedState } from "metabase-types/store";
+import { DEFAULT_EMBED_OPTIONS } from "metabase/redux/embed";
 
-export const createMockEmbedOptions = (opts?: Partial<EmbedOptions>) => ({
+const createMockEmbedOptions = (opts?: Partial<EmbedOptions>) => ({
+  ...(DEFAULT_EMBED_OPTIONS as EmbedOptions),
```

Comment:
Use the actual default embed option to reflect the actual app behavior.

Reply:
`metabase-types` shouldn't rely on the app. It's an oversight that this change is not automatically  rejected by ESLint.

Reply:
What can be done instead:
- First of all, fix the issue with the types. You shouldn't need to cast `DEFAULT_EMBED_OPTIONS` to `EmbedOptions`. Either the type or the constant should be fixed.
- Pass `DEFAULT_EMBED_OPTIONS` when creating the mock state in the test (if you want to do it that way).

Reply:
1. Agreed with fixing the type, I don't remember why I need to do that
2. Let me think about this a bit more. I think having a useful default value in the state could make DX better since what we're mocking is making much more sense than just an empty state.

Reply:
re: 2. I decided to follow your suggestion because I don't want to make this file an only anomaly. Also, people would have different notions of useful defaults, leaving it blank seems a good idea